src/workspace-data/Data.js

```diff
@@ -632,7 +637,14 @@ export const WorkspaceData = _.flow(
       }
     };
 
-    const loadMetadata = () => Promise.all([loadEntityMetadata(), loadSnapshotMetadata(), refreshRunningImportJobs(), loadWdsSchema()]);
+    const loadAzureSnapshotMetadata = async () => {
+      setSnapshotMetadataError(false);
+    };
```

Comment:
very minor nit: the two method names "azureLoadEntityMetadata" and "loadAzureSnapshotMetadata" are not consistent in their ordering of "azure" and "load"